README.rst

```diff
@@ -31,7 +31,36 @@ You can use ``pip`` to install codespell with e.g.::
 Usage
 -----
 
-Check usage with ``codespell -h``. There are a few command line options.
+For more in depth info please check usage with ``codespell -h``. 
+
+Some noteworthy flags::
+
+    codespell -w, --write-changes
+
+The -w flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is equated with doing a dry run. Recommended to run this with the ``-i`` or ``--ineractive`` flag.:: 
```

Comment:
is equated -> is the same as

Recommended to -> It is recommended to

--ineractive -> --interactive

Reply:
👍